cmd/storage-errors.go

```diff
@@ -143,7 +143,7 @@ func osErrToFileErr(err error) error {
 	if osIsPermission(err) {
 		return errFileAccessDenied
 	}
-	if isSysErrNotDir(err) {
+	if isSysErrNotDir(err) || isSysErrIsDir(err) {
```

Comment:
Maybe it is better to return errFileAccessDenied in case of isSysErrIsDir() ? I think we used to have this way before. Why this change anyway ?

Reply:
That is only during the write phase @vadmeste - in the read phase, we always return `errFileNotFound`